### ISSUE TITLE
Transparent gallery icons missing contact against white backgroun

### ISSUE THREAD

**Issue by b-meson** [lucyparsons/OpenOversight#352]
<img width="457" alt="screen shot 2018-01-17 at 7 41 16 pm" src="https://user-images.githubusercontent.com/3100220/35079804-8ba83a3e-fbbe-11e7-9c2d-aca02d79a307.png">
 The above icons (the filled in circles) do not appear on certain images in the gallery.

**Comment by furfreemi** [comment]
think this could be addressed by simply making them light gray all the time? or is the white strongly preferred if they're overlaying the image, then only have light gray if they're on that white background due to a smaller image?
![image](https://user-images.githubusercontent.com/9063936/35186734-f481e5ee-fde6-11e7-9d3e-cbc0adc16d2f.png)

**Comment by b-meson** [comment]
@furfreemi the last time we had a demo of OO someone from the community said "it was not clear that it was a gallery you can search through". So this might be a bigger UI/UX issue that redoing the colors. Thoughts?

**Comment by furfreemi** [comment]
ahh i see. hmm. 
one question- is there historical context behind pagination into multiple galleries at beyond 20 images? load time or? didn't look at the code around that yet, apologies if that would answer my question
![image](https://user-images.githubusercontent.com/9063936/35190272-528b3642-fe2b-11e7-9bfd-1df45e0ae0a4.png)

**Comment by furfreemi** [comment]
notes about the issue:
 - not obvious that there's multiple images that are intended to be looked through

potential reasons:
- no concept/representation of multiple images when looking at the page
 -- dot indicators aren't consistently visible
 -- the user only ever sees one image on this page
- lack of purpose and clear instructions on page
- unclear cta around 'that's the officer!'
 -- where will this take the user?

other things i noticed while playing with it:
 - not able to pause slideshow
 - off-by-one error when clicking the dots to try to move to a particular image
 - often end up blindly click to try to go back to a specific image, as it's unclear what each dot actually represents

-------------------

### wip 1
if there's potentially a ton of images and don't want to bombard the user, could put previews at the bottom so it's easier to focus on the main image. could lead to people not scrolling down though and ending up with the same impression of the page as before. maybe a count above the image (displaying officer x of y) could help with that

![oo_preview_bottom](https://user-images.githubusercontent.com/9063936/35191124-89251aa0-fe40-11e7-9e08-f857c03a2595.jpg)


-------------------
### wip 2
how many images are usually displayed to the user? would something like this get too overwhelming in terms of having image previews up top? not loving the light blue box but seems like some grouping around the current officer section would be good to have:

![oo_preview_top](https://user-images.githubusercontent.com/9063936/35191125-89330d7c-fe40-11e7-9376-ed4370d8338e.jpg)

**Comment by b-meson** [comment]
IIRC the first release of OO had no pagination so it would load the entire gallery. Then I think we moved to paginate to 20 images. After that I think we then changed to the carousel wheel display. I think that's the historical context.  

> not obvious that there's multiple images that are intended to be looked through

Yeah, so I feel like this one should say "returning any _potential_ matches".

> unclear cta around 'that's the officer!'

Great point as well.

**Comment by r4v5** [comment]
Yep, the historical context is our original release's code was doing some inefficient things (see #97) and when people queried things with a large potential response ("all police officers in Chicago age 18 to 65, don't care for all the values" for instance) the thread would exceed the 90 second timeout gunicorn has for responses, and return an ugly nginx 500 page to the user. #16 and #95 captured the pain points; #101 and #97 were shorter- and longer-term fixes for the back-end performance issue.

I'm not sure that we can ever eliminate pagination entirely without some Front-End Magic, though, because wide queries have the potential to return a hilarious number of officers, all of which have to be rendered in the DOM, and we don't want to crush people's machines rendering 20,000 images.

**Comment by furfreemi** [comment]
hahaha well that's very good to know, thank you both for the info and additional references!
